### DISCUSSION TITLE
[Quick question] Surface Book 2 Open source Nvidia Driver

### DISCUSSION THREAD

**Discussion by 2ndBillGates** [linux-surface/linux-surface#577] [Q&A] [answered]
I just installed Nvidia-dkms without Linux-Surface Kernel and found some application not running well.

Can I install nouveau driver then install Surface Kernel?

**Comment by 2ndBillGates** [comment]
Also I want to put my bootloader into SD card to boot my LUKS root parition, to improve security, avoid evil maid.

**Comment by qzed** [comment, marked as answer]
You might want to make sure that the Nvidia GPU isn't used to render the desktop and only use that for rendering graphics intense applications (there might be some Xorg configuration necessary for that). The dGPU is unfortunately known to have bad performance on Linux without connected AC. Also it will consume quite a bit more battery than the integrated one.

I don't think you'll get any better results with nouveau, but feel free to try. I haven't tested that yet, so can't say how well that driver works for that generation of cards (or the one in the SB2 in particular). The driver itself should be included in the kernel (and work on cards it supports) so all you need to do is probably some configuration.
